public/css/dark-theme.css

```diff
@@ -0,0 +1,188 @@
+    -ms-transition: all 1s ease;
+    transition: all 1s ease;
+}
+
+/* END SMOOTH TRANSITION */
+
+html, body, input, select, button {
+    color: var(--font-color);
+    background-color: var(--bg-color);
+}
+
+.table, input:focus, .form-control:focus {
+    color: var(--font-color) !important;
+    background-color: var(--bg-color) !important;
+}
+
+.page-item:not(.active) .page-link {
+    background-color: var(--bg-color);
+}
+
+.page-item.disabled .page-link {
+    color: #6c757d;
+}
+
+.table td, .table th {
+    background-color: var(--bg-color-strongest);
+    border-color: var(--border-color);
+    color: var(--font-color-1);
+}
+
+.items-table::before {
+    background-image: var(--background-image-gradient-before);
+}
+
+.breadcrumb {
+    list-style-image: initial;
+    background-color: var(--bg-color-1);
+}
+
+.form-control {
+    background-color: var(--bg-color);
+    /* border-color: var(--border-color); */
+    color: var(--font-color-1);
+}
+
+.form-control:disabled, .form-control[readonly] {
+    background-color: var(--bg-color-1);
+    color: var(--font-color-2);
+}
+
+/* BEGIN VARS */
+
+:root {
+    --tabs-border-bottom-color: #dee2e6;
+    --border-color-dim: rgba(139, 130, 117, 0.13);
+}
+
+[data-theme="dark"] {
+    --tabs-border-bottom-color: rgb(50, 54, 56);
+    --border-color-dim: rgba(139, 130, 117, 0.13);
+}
+
+/* END VARS */
+
+input, .form-control {
+    border-color: var(--border-color-1);
+}
+
+/* NAV TABS */
+
+.nav-tabs {
+    border-bottom-color: var(--tabs-border-bottom-color);
+}
+
+.nav-tabs .nav-link {
+    border-color: transparent;
+}
+
+.nav-tabs .nav-link:focus, .nav-tabs .nav-link:hover {
+    border-color: var(--tabs-border-bottom-color);
+}
+
+.nav-tabs .nav-link.disabled {
+    /* color: rgb(159, 152, 141); */
+    background-color: transparent;
+    border-color: transparent;
+}
+
+.nav-tabs .nav-item.show .nav-link, .nav-tabs .nav-link.active {
+    /* color: rgb(185, 180, 172); */
+    color: var(--font-color-2);
+    background-color: var(--bg-color);
+    border-color: var(--border-color);
+}
+
+.table-bordered td, .table-bordered th {
+    border-color: var(--border-color);
+}
+
+.list-group-item {
+    background-color: var(--bg-color-strongest);
+    border-color: var(--border-color-dim)
+}
+
+.footer {
+    background-color: var(--bg-color);
+    border-color: var(--border-color);
+}
+
+a {
+    color: var(--link-color);
+}
+
+/* **************************** */
+
+/* json formatter */
+
+/* **************************** */
+
+/* BEGIN VARS */
+
+:root {
+    --json-formatter-string: green;
+}
+
+[data-theme="dark"] {
+    --json-formatter-string: #03bf03;
+}
+
+/* END VARS */
+
+.json-formatter-row, .json-formatter-row a, .json-formatter-row a:hover {
+    color: rgb(242, 240, 238);
+    text-decoration-color: initial;
+}
+
+.json-formatter-row .json-formatter-string, .json-formatter-row .json-formatter-stringifiable {
+    color: var(--json-formatter-string) !important;
```

Comment:
Some type of data is pretty much unreadable due to poor contrast:

![Screenshot 2021-10-03 at 20 06 34](https://user-images.githubusercontent.com/153197/135766112-cc3e62f5-e4c5-4529-ba07-3829db8bb1a0.png)

This one is a field that has an object value

Reply:
Solved with a neat design with colors in the cyan spectrum:

<img width="693" alt="Screenshot 2021-10-05 at 13 02 57" src="https://user-images.githubusercontent.com/5558911/136003036-13875635-600e-4793-8e37-e647f612dbde.png">